stable/vpa/templates/_helpers.tpl

```diff
@@ -40,6 +40,18 @@ helm.sh/chart: {{ include "vpa.chart" . }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- if .Values.podLabels }}
+{{ toYaml .Values.podLabels }}
+{{- end }}
+{{- if .Values.recommender.podLabels }}
+{{ toYaml .Values.recommender.podLabels }}
+{{- end }}
+{{- if .Values.updater.podLabels }}
+{{ toYaml .Values.updater.podLabels }}
+{{- end }}
+{{- if .Values.admissionController.podLabels }}
+{{ toYaml .Values.admissionController.podLabels }}
+{{- end }}
```

Comment:
If I set .Values.updater.podLabels, won't they end up on every deployment with this setup? I think the component-specific ones need to be added to the individual deployment templates

Reply:
Thanks for the catch! Pushed an update :)